webrender/src/device/gfx.rs

```diff
@@ -2170,6 +2187,116 @@ impl<B: hal::Backend> Device<B> {
             (device, queues.take(id).unwrap())
         };
 
+        #[cfg(feature = "headless_gfx")]
+        let (
+            surface_format,
+            depth_format,
+            render_pass,
+            framebuffers,
+            framebuffers_depth,
+            frame_depths,
+            frame_images,
+            viewport,
+            frame_count,
+        ) = {
+
+            let surface_format = ImageFormat::BGRA8;
+            let depth_format = hal::format::Format::D32Sfloat;
+            let render_pass = Device::create_render_passes(
+                &device,
+                hal::format::Format::Bgra8Unorm,
+                depth_format
+            );
+
+            let extent = hal::window::Extent2D {
+                width: window_size.0 as u32,
+                height: window_size.1 as u32,
+            };
+            let mut frame_depths = Vec::new();
+            let (frame_images, framebuffers, framebuffers_depth) = {
+                let extent = hal::image::Extent {
+                    width: extent.width as _,
+                    height: extent.height as _,
+                    depth: 1,
+                };
+                let mut images = Vec::new();
```

Comment:
These images are needed for creating the cores below. I think we can skip creating them, and just simply call `ImageCore::create`. See https://github.com/szeged/webrender/blob/master/webrender/src/device/gfx.rs#L592 in `Image::new`